packages/apputils-extension/style/contextualhelp.css

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+:root {
```

Comment:
Since these are private we can move them to `.jp-ContextualShortcut-Table` (proposed new class) and `jp-ContextualShortcut-Key` respectively.

Please see https://github.com/jupyterlab/jupyterlab/issues/14167 for context.

Reply:
@krassowski Thanks for the comments. I have changed `--jp-private-shortcuts-table-padding-****`to `--jp-private-shortcuts-row-padding-****` and finally did not need to introduce the new class `.jp-ContextualShortcut-Table`but used already existing one `.jp-ContextualShortcut-TableRow`. I moved the classes out of root section.